matRad_calcQualityIndicators.m

```diff
@@ -125,10 +127,5 @@
     end
```

Comment:
Eh I figure we need a case when the last structure of cst is empty (just ran into that rare one), cause then qi is not equal in size with cst which is confusing.
```
        listOfFields = fieldnames(QI);
        for j = 1:numel(listOfFields)
            QI(runVoi).(listOfFields{j}) = [];
        end
```


(We may also think of adding the name of the structure for security reasons.)